calculator_test.go

```diff
@@ -6,20 +6,17 @@ import (
 	"testing"
 )
 
-type testCase struct {
-	a, b        float64
-	want        float64
-	name        string
-	errExpected bool
-}
-
 func TestAddSubtractMultiply(t *testing.T) {
 	t.Parallel()
-	addTestCases := []testCase{
+	addTestCases := []struct {
```

Comment:
Actually, since the struct is the same for Add, Subtract, and Multiply test cases, we could use one single slice of cases for all three functions, couldn't we? We'd need some way to specify _in_ the test case which function to call. But we can do that: we can add a struct field whose type is `func ...` (some exact function signature). And you know the signature, because it's the same for all three functions!